doc/massdeploymentcommandline.rst

```diff
@@ -0,0 +1,30 @@
+It is possible to perform mass deployment of the Nextcloud desktop client by passing certain command-line parameters from the deployment step to Nextcloud desktop client executable after the initial setup.
+This will allow desktop client to generate a config (.cfg) file that will be used during subsequent launches.
+A config file will have a corresponding account written into it similar to if you have added it manually via the desktop client's UI.
+The desktop client will exit with code 0 if account has been added successfuly, or 1 in case of failure.
+Detailed failure message is printed to the desktop client logs.
+
+The following parameters are supported:
+
+``--userid``
+        (required) userId (username as on the server) to pass when creating an account via command-line.
+
+``--apppassword``
+        (required) appPassword to pass when creating an account via command-line (see the ``login-flow`` section in server documentation on how to generate the app password).
+
+``--localdirpath``
+        (optional) path where to create a local sync folder when creating an account via command-line. If skipped, then default local sync folder path (/home/<userid>/Nextcloud<n> for Linux/mac or C:/<userid>/Nextcloud<n> for Windows) will be generated by desktop client.
+        
+``--isvfsenabled``
+        (optional) whether to set a VFS or non-VFS folder (1 for 'yes' or 0 for 'no') when creating an account via command-line. Default is 0.
+
+``--remotedirpath``
+        (optional) path to a remote subfolder when creating an account via command-line. e.g. If the server has folders "/Photos", "/Documents", "/Music" you can pass "/Music" and then this folder will get set up as remote root.
+
+``--serverurl``
+        (required) a server URL to use when creating an account via command-line. (NOTE: There is another parameter supported by Nextcloud desktop client ``--overrideserverurl`` but it SHOULD NOT be used here as it is intended for setup via UI with wizard)
```

Comment:
can you also add the same for `--overridelocaldir` ?

Reply:
@mgallien done in `doc/wizardaccountsetupcommandline.rst`